src/celeritas/em/interactor/AtomicRelaxation.hh

```diff
@@ -143,17 +141,25 @@ AtomicRelaxation::operator()(Engine& rng)
         if (vacancy_id.get() >= shells.size())
             continue;
 
-        // Sample a transition (TODO: refactor to use Selector but with
-        // "remainder")
```

Comment:
We also have a TODO for this in the photoelectric interactor, not sure if that's a straightforward update too?